scss/bootstrap/variants/_outlined-buttons.scss

```diff
@@ -1,6 +1,4 @@
-@import 'bootstrap/mixins/button-outline-variant';
```

Comment:
What if I decide, at some point, to import only `outlined-buttons` partial in some theme? To me, it sound that all I would need to do is import only one file, I should not care about mixins it uses. Because you remove that, I then will not be able to simply import the partial, I will need to know which mixins it uses and import them as well.

Reply:
you're right and I wasn't sure if to remove them or to simply convert the paths to be relative.

in the end I decided to remove them for consistency: sass projects (or at least bootstrap's) have a flat structure, you load all the necessary (mixins, functions, variables) in the main sass file before importing the partials with actual css in them. Otherwise we would end up re-importing the same mixins and functions and variables in every single partial. 

Even the only import I left in the file, `@import '../overrides/variables'`, was something that i wanted to remove, but couldn't due to variable values conflicts between Shoreditch and the SSP (basically the partial needs the variables value in Shoreditch, even if it's imported in the SSP. Something that smells of bad architecture, but that cannot investigate in the limited scope of the ticket)